src/github/push.ts

```diff
@@ -33,15 +32,13 @@ export const pushWebhookHandler = async (context: WebhookContext, jiraClient, _u
 		gitHubInstallationId
 	});
 
-	if (await booleanFlag(BooleanFlags.CONFIG_AS_CODE, false, jiraClient.baseURL)) {
-		if (subscription) {
-			const modifiedFiles = context.payload?.commits?.reduce((acc, commit) =>
-				([...acc, ...commit.added, ...commit.modified, ...commit.removed]), []);
-			// TODO: this call must be updated to support GitHub Server events
-			await updateRepoConfig(subscription, payload.repository.id, getInstallationId(gitHubInstallationId), modifiedFiles);
-		} else {
-			context.log.warn("could not load user config because subscription does not exist");
-		}
+	if (subscription) {
+		const modifiedFiles = context.payload?.commits?.reduce((acc, commit) =>
+			([...acc, ...commit.added, ...commit.modified, ...commit.removed]), []);
```

Comment:
This is the spot that crashed when removing the FF, because `commit.added` wasn't set